.github/workflows/daily.yml

```
@@ -19,6 +19,45 @@ on:
         default: 'master'
 
 jobs:
+  tests-with-elle:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install build dependencies
+        run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen
+      - name: Build
+        run: |
+          mkdir build && cd build
+          cmake .. -DPYTEST_OPTS="--redis-executable=redis/src/redis-server --elle-cli=elle-cli/target/elle-cli-0.1.4-standalone.jar --elle -v"
```

```suggestion
          cmake .. -DPYTEST_OPTS="--redis-executable=redis/src/redis-server --elle -v"
```

If this is same with the default path, let's delete it to keep here a bit simpler